app/Repositories/PhotoRepository.php

```diff
@@ -42,6 +42,9 @@ public function __construct(AWS $aws, Registrar $registrar)
      */
     public function create(array $data, $signupId)
     {
+        // Set quantity and why_participated to null - we don't want this to live on the post_photo event.
+        $data['quantity'] = null;
+        $data['why_participated'] = null;
```

Comment:
I kind of like the idea of just not passing these to create the event, maybe using something like this? https://laravel.com/docs/5.2/helpers#method-array-except